PayPalNativeCheckout/src/test/java/com/paypal/android/checkout/PayPalClientTest.kt

```diff
@@ -66,8 +80,8 @@ class PayPalClientTest {
             PayPalCheckout.startCheckout(any())
         } just runs
 
-        sut = PayPalClient(mockApplication, coreConfig, mockReturnUrl, api)
-        sut.startCheckout(generateRandomString())
+        sut = PayPalCheckoutClient(mockApplication, coreConfig, mockReturnUrl, api)
+        sut.startCheckout(mockk())
```

Comment:
what does the second K mean?

Reply:
it stand for `Kotlin`. It is a mocking library specficially designed for kotlin, called MockK